src/global/variables.scss

```diff
@@ -1,5 +1,2 @@
-@import './node_modules/lyne-design-tokens/dist/scss/_variables.scss';
-
-:host {
-  --app-primary-color: blue;
-}
+//@import './node_modules/lyne-design-tokens/dist/scss/_variables.scss';
```

Comment:
This one you want to keep for now, just in case?

Reply:
Yes. As we discussed, we most probably will have variables that we need to define in scss. Can't think of a specific example now, but i'd like to keep it in case rather then to rebuild this (and the change in stencil.config) later on.